### ISSUE TITLE
Steam on Linux: the entire client is totally inaccessible for blind users

### ISSUE THREAD

**Issue by francescotissera1211** [ValveSoftware/steam-for-linux#9911]
#### Your system information

* Steam client version (build number or date): I have no idea, since the client cannot be navigated with a screen reader.
* Distribution (e.g. Ubuntu): ArchLinux
* Opted into Steam client beta?: [Yes/No]: no
* Have you checked for system updates?: [Yes/No]: yes
* Steam Logs: [generate by running this command in a terminal `tar -zcvf ~/Desktop/steam-logs.tar.gz ~/.steam/steam/logs`]
* GPU: Nvidia/Intel/AMD: AMD

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen. Please link any large code pastes as a [Github Gist](https://gist.github.com/)
As soon as installation finishes and the client is opened, said client's interface is not accessible at all with the Orca screenreader.
Tabbing around, arrowing around or trying to navigate in any way results in no speech from the screenreader.
This results in a blind user not being able to use the steam client at all on Linux to play both native and proton compatible games.
#### Steps for reproducing this issue:

1. install Steam with sudo pacman -S steam.
2. turn on the screen reader and open the client.
3. try to navigate around. You might get the steam runtime content updating screen to be seen by Orca, but that will be it.

**Comment by kisak-valve** [comment]
Noted on #10178, this improved for a while, but functionally regressed before the October 25th Steam client update.

**Comment by lilmike** [comment]
Yes, the public beta was accessible for a while, but now regressed to being inaccessible.

-Michael.

**Comment by RealAmethyst** [comment]
Tested this extensivly these last few days. I\ve tried running this with steam_runtime=0 and other veriables, with no result. The flatpak version of steam is fully accessible using Orca, so I have no idea what\s going on there. Could someone check what\s different and port it over to the native client_ I\m running Arch btw.
